components/telemetry-operator/controller/logpipeline/controller.go

```diff
@@ -131,16 +171,14 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	}
 
 	if changed {
-		log.V(1).Info("Fluent Bit configuration was updated. Restarting the DaemonSet")
+		log.Info("Fluent Bit configuration was updated. Restarting the DaemonSet")
```

Comment:
This PR reaplces `log.Info` by `log.V(1).Info` for most of the occurrences (I guess for consistency?). Is there any reason to do it vice-versa in this line?

Reply:
Yeah, most of the info logs created too much noise, so they were demoted to debug. However, this one is good to have with the info level in order to see which reconciliation actually triggered a restart.